tests/_utils/config.py

```diff
@@ -102,16 +102,28 @@ def test_files(self) -> tuple[Path, ...]:
         if self.examples:
             examples = (
                 path
-                for path in Path("examples").glob("*.py")
+                for path in Path(self.root_dir.joinpath("examples")).glob(
```

Comment:
The explicit `Path` is still superfluous since `self.root_dir` is already a `Path`
```suggestion
                for path in self.root_dir.joinpath("examples").glob(
```

Reply:
Without `Path`, it would raise AttributeError: 'PurePosixPath' object has no attribute 'glob'. 
As self.root_dir.joinpath("examples") is 'PurePosixPath', `Path` is necessary here.

Reply:
I'm not sure why it is currently a pure path, I think the `root_dir` property should change to return a `Path` instead.

Reply:
Done. Changed root_dir to return a Path instead of PurePath.